src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Repositories/Interfaces/IAmPartyRepository.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Altinn.Authorization.Core.Models.Party;
+
+namespace Altinn.AccessManagement.Core.Repositories.Interfaces
+{
+    /// <summary>
+    /// Interface for party repository
+    /// </summary>
+    public interface IAmPartyRepository
+    {
+        /// <summary>
+        /// Get party by party uuid
+        /// </summary>
+        public Task<MinimalParty> GetByUuid(Guid partyUuid, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Get by party id
+        /// </summary>
+        public Task<MinimalParty> GetByPartyId(int partyId, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Get Party by org number
+        /// </summary>
+        public Task<MinimalParty> GetByOrgNo(string orgNo, CancellationToken cancellationToken = default);
```

Comment:
Maybe use `OrganizationIdentifier` instead of `string`? Same with the `ByPersonNo` below.